gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/ModuleArgs.java

```diff
@@ -40,6 +41,7 @@
  * plugin <a href="https://github.com/palantir/gradle-baseline/pull/1944">gradle-baseline#1944</a>.
  */
 final class ModuleArgs {
+    private static final Logger log = Logging.getLogger(ModuleArgs.class);
```

Comment:
does gradle route this tot he project logger here? not sure how it would :S

Reply:
Unfortunately there's no injectable logger you can use. This is the only real ways I found. https://github.com/gradle/gradle/issues/2678